tensorboard/compat/tensorflow_stub/io/gfile.py

```diff
@@ -115,6 +117,9 @@ def read(self, filename, binary_mode=False, size=None, continue_from=None):
         """
         mode = "rb" if binary_mode else "r"
         encoding = None if binary_mode else "utf8"
+        if not exists(filename):
+          raise errors.NotFoundError(
```

Comment:
+2 spaces here to stick with 4 space indent consistently in this file?

Reply:
good catch, thanks